### PR TITLE
Feature/PTS-1238 polly-docs website header update

### PR DESCRIPTION
**Pull request by badal-pareta**
### Type of Pull Request

- [x] 💡 Feature
- [ ] 🛠 Bug Fix
- [ ] 📁 Refactor
- [ ] ⚡️ Optimization

### Feature / Fix Title
Added backlink to main site elucidata.io & Request demo button in polly-docs header
### Description
Added backlink to main site elucidata.io & Request demo button in polly-docs header
Custom header template to override the theme header.

### JIRA Ticket IDs
PTS-1238

### Impacted Areas
Header for documentation

### Done by
@badal-pareta 

### Screenshots (if appropriate)
Screenshot 2024-04-11 at 4 21 37 PM

### CODE REVIEW

File: docs/assets/stylesheets/custom.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.md-search__form {
+    border-radius: 24px;
```
Comment (yagnesh-patil) [inline docs/assets/stylesheets/custom.css:2]: Use rem units

File: docs/assets/stylesheets/custom.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.md-footer-meta {
+    display: none;
+}
+
+.demo-button {
+    align-items: center;
+    background-color: #CB6E00;
```
Comment (yagnesh-patil) [inline docs/assets/stylesheets/custom.css:11]: Create css variable for now in this file.

Usage example
```css 
:root {
  --color-secondary: #CB6E00;
}

.demo-button {
  background-color: var(--color-secondary);
}
```

File: docs/assets/stylesheets/custom.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    border-radius: 24px;
+    color: #FFFFFF;
+    cursor: pointer;
+    font-family: Inter, sans-serif;
```
Comment (yagnesh-patil) [inline docs/assets/stylesheets/custom.css:15]: How are we importing the font?

File: overrides/partials/header.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
+{#-
+    This file was automatically generated - do not edit
```
Comment (yagnesh-patil) [inline overrides/partials/header.html:2]: Can we remove this comment?

Comment (badal-pareta) [reply overrides/partials/header.html:2]: We can, its better to keep it as it is, to keep  in track with the partial override from the material's template, when theme is updated

File: docs/assets/stylesheets/custom.css (hunk 3 above)
Comment (badal-pareta) [reply docs/assets/stylesheets/custom.css:15]: We are not importing fonts for now.  Can be taken up as an addon with UI updation like suggested

File: docs/assets/stylesheets/custom.css (hunk 1 above)
Comment (badal-pareta) [reply docs/assets/stylesheets/custom.css:2]: Will have to explore the theme to check root element

File: docs/assets/stylesheets/custom.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
+:root {
+    --color-orange-dark: #CB6E00;
+    --color-orange-light: #f78e12;
+    --color-white: #FFFFFF;
+  }
```
Comment (yagnesh-patil) [inline docs/assets/stylesheets/custom.css:5]: additional space before }

File: docs/assets/stylesheets/custom.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+.demo-button:hover {
+    background-color: var(--color-orange-light);
+}
+
+.d-flex {
+    display: flex;
+}
+
+.gap-1 {
+    grid-gap: 0.4rem;
```
Comment (yagnesh-patil) [inline docs/assets/stylesheets/custom.css:36]: 0.5rem for 8px

**Review by yagnesh-patil — APPROVED** [review]

File: docs/assets/stylesheets/custom.css (hunk 6 above)
Comment (badal-pareta) [reply docs/assets/stylesheets/custom.css:36]: root size 20px

Comment (badal-pareta) [reply docs/assets/stylesheets/custom.css:36]: Screenshot 2024-04-11 at 10 12 39 PM

**Pull request merged**